src/pages/LetterWritePage/components/LetterReceiverBottomSheet.tsx

```diff
@@ -0,0 +1,106 @@
+import { useState } from 'react';
+import { useFormContext } from 'react-hook-form';
+import { css } from '@emotion/react';
+import COLORS from '@/constants/colors';
+import textStyles from '@/styles/textStyles';
+import BottomSheet from '@/components/BottomSheet';
+import { ArrowClockWise } from '@/assets/icons';
+import { BottomSheetProps } from './LetterWriteContent';
+import { AgeSlider, GenderSelect, ConcernSelect } from '.';
+
+const LetterReceiverSelect = ({
+  isBottomSheetOpen,
+  toggleBottomSheet,
+}: BottomSheetProps) => {
+  const { setValue } = useFormContext();
+
+  const [age, setAge] = useState([15, 40]);
+  const [gender, setGender] = useState('');
+  const [concern, setConcern] = useState('');
```

Comment:
ask;
이 데이터는 react-hook-form 으로 관리가 되는 것으로 이해했어요!! 그래서 `watch` 옵션으로 꺼내와서 사용할 수 있을 것 같은데 이 컴포넌트에서 추가적인 지역 상태로 관리되고 있는 부분이 궁금해요 🧐
(여러 상태가 같은 값을 가리켜야 하는 형태여서 동기화가 깨지기 쉬운 부분일 것 같아서요!)

https://ko.react.dev/learn/choosing-the-state-structure#avoid-redundant-state

```suggestion
  const age = watch('age');
  const gender = watch('gender');
  const concern = watch('concern');
```

Reply:
저는 `BottomSheet` 에서 무조건 **확인** 버튼을 눌러야 폼에 값이 들어가도록 구현했어요! 

처음에 이런식으로 바로 폼에 값이 들어가게 했었는데 이 상태에서는 `BottomSheet` 의 **확인**, **닫기** 버튼의 경계가 사라지더라구요🥲 **확인**을 누르나 **닫기**를 누르나 무조건 폼에 들어가고, **닫기** 를 눌렀다고 폼에 등록된 내용을 다 지울 수는 없으니깐요
<img width="519" alt="image" src="https://github.com/dnd-side-project/dnd-10th-4-frontend/assets/98106371/1020c4b1-7e15-400c-b013-f4806ab6e272">

그래서 `useState` 를 사용해 각각 상태를 관리했습니다!

**확인** 버튼을 누르면 각 입력받은 나이, 고민, 성별 값이 **form** 의 age, concern, gender 로 들어갑니다.

```tsx
  const onCompleteButtonClick = () => {
    setValue('age', age);
    setValue('concern', concern);
    setValue('gender', gender);
    toggleBottomSheet(false)();
  };
```

**Refresh** 버튼을 누르면 사용자가 선택한 값과 **form** 에 들어간 값이 초기화 됩니다.

```tsx
  const onRefreshIconClick = () => {
    setValue('age', []);
    setValue('gender', '');
    setValue('concern', '');
    setAge([15, 40]);
    setGender('');
    setConcern('');
    setIconRotation((prev) => prev + 360);
  };
```